Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:bionic
+
+ENV TZ=America/Chicago
+RUN    ln --symbolic --no-dereference --force /usr/share/zoneinfo/$TZ /etc/localtime \
+    && echo $TZ > /etc/timezone
+
+RUN apt update && apt upgrade -y
+
+RUN apt install --yes                                                             \
+        autoconf bison clang-6.0 cmake curl flex gcc git libboost-test-dev        \
+        libffi-dev libgmp-dev libjemalloc-dev libmpfr-dev libtool libyaml-cpp-dev \
+        libz3-dev make maven opam openjdk-8-jdk pkg-config z3 zlib1g-dev
+
+RUN update-alternatives --set java /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java
+
+RUN curl -sSL https://get.haskellstack.org/ | sh
+
+ARG USER_ID=1000
+ARG GROUP_ID=1000
+RUN    groupadd --gid $GROUP_ID user                                        \
+    && useradd --create-home --uid $USER_ID --shell /bin/sh --gid user user
+
+USER $USER_ID:$GROUP_ID
+
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain 1.28.0
+
+RUN    cd /home/user                                                          \
+    && git clone 'https://github.com/kframework/k' --branch=nightly-0f3835d3a \
```

Comment:
why are we even setting up opam on this particular dockerfile, anyway? it shouldn't be at all necessary for this repo, and setting it up this way instead of via an ADD is very fragile.

Reply:
I disagree that it's fragile, but I agree it's not needed. I'll pull it out.

Reply:
Done, I also pulled out the `rustup`.